spec.bs

```diff
@@ -2016,3 +2015,27 @@ achieve the outcomes described in the above explanatory content.
   /fenced-frame/permission-geolocation.https.html
   /fenced-frame/permission-notification.https.html
 </wpt>
+
+<h3 id=local-network-access-monkeypatch>Local Network Access</h3>
```

Comment:
It looks like the spec is maybe being renamed back to PNA (see https://github.com/WICG/local-network-access/issues/91#issuecomment-1494704528 and https://github.com/WICG/local-network-access/issues/91#issuecomment-1540157575), can you update your PR accordingly?